fluster/decoders/iso_mpeg2_video.py

```diff
@@ -0,0 +1,88 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2024, Fluendo, S.A.
+#  Author: Rubén Sánchez <rsanchez@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation, either version 3
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library. If not, see <https://www.gnu.org/licenses/>.
+import glob
+import os
+
+from fluster.codec import Codec, OutputFormat
+from fluster.decoder import Decoder, register_decoder
+from fluster.utils import run_command
+
+
+@register_decoder
+class ISOMPEG2VDecoder(Decoder):
+    """ISO MPEG2 Video reference decoder implementation"""
+
+    name = "ISO-MPEG2-VIDEO"
+    description = "ISO MPEG2 Video reference decoder"
+    codec = Codec.MPEG2_VIDEO
+    binary = "mpeg2decode"
+    is_reference = True
+
+    def decode(
+        self,
+        input_filepath: str,
+        output_filepath: str,
+        output_format: OutputFormat,
+        timeout: int,
+        verbose: bool,
+        keep_files: bool,
+    ) -> str:
+        """Decodes input_filepath in output_filepath"""
+        base_dir = os.path.splitext(os.path.basename(os.path.normpath(output_filepath)))[0]
+        temp_dir = os.path.join(os.path.join(os.path.dirname(output_filepath), base_dir))
+        os.makedirs(temp_dir, exist_ok=True)
+        output_filepath += ".yuv"
```

Comment:
`output_filepath` must not be changed, otherwise the caller of the function will not know that the path changed.